lib/endpoints/class-wp-rest-comments-controller.php

```diff
@@ -953,6 +953,9 @@ public function get_item_schema() {
 					'description'     => __( 'The content for the object.' ),
 					'type'            => 'object',
 					'context'         => array( 'view', 'edit', 'embed' ),
+					'arg_options' => array(
+						'sanitize_callback' => null,
+					),
```

Comment:
Agreed that this should have a comment, maybe something simpler like "Note: sanitization implemented in callback"?